array.c

```diff
@@ -5833,14 +5623,7 @@ recursive_cmp(VALUE ary1, VALUE ary2, int recur)
  *  - Returns 0 if +array+ and +other_array+ are the same size:
  *      [0, 1, 2] <=> [0, 1, 2] # => 0
  *
- *  ---
- *
- *  Argument +other_array+ may be an
- *  {Array-convertible object}[doc/implicit_conversion_rdoc.html#label-Array-Convertible+Objects]:
- *    require 'csv'
- *    [] <=> CSV::Row.new([], []) # => 0
- *
- *  Returns +nil+ if +other_array+ is not an Array-convertible object:
+ *  Returns +nil+ if +other_array+ is not an \Array.
  *    [] <=> 0 # => nil
```

Comment:
We may want to delete this example, since it doesn't add new information.

Reply:
We shall delete this example, but later, in the course of eliminating low-information 1-line examples.